packages/plugins/rpc-manager/src/lib/eth_sendRawTransaction.ts

```diff
@@ -0,0 +1,138 @@
+    // eth_sendRawPrivateTransaction)
+    const originalPayload = params.originalRequest?.params[0];
+    const privateFor = originalPayload?.privateFor;
+    const privateFrom = originalPayload?.privateFrom;
+    if (!privateFor && !privateFrom) {
+      return callback(null, params);
+    }
+
+    const account = accounts.find((acc) => Web3.utils.toChecksumAddress(acc.address) === Web3.utils.toChecksumAddress(originalPayload.from));
+    if (!account?.privateKey) {
+      return callback(null, params);
+    }
+
+    // manually send to the node in the response
+    params.sendToNode = false;
+    callback(null, params);
+  }
+
+  private async ethSendRawTransactionResponse(params: any, callback: Callback<any>) {
+    if (params.request.method !== blockchainConstants.transactionMethods.eth_sendRawTransaction) {
+      return callback(null, params);
+    }
+
+    const accounts = await this.accounts;
+    if (!(accounts && accounts.length)) {
+      return callback(null, params);
+    }
+
+    // Only handle quorum requests that came via eth_sendTransaction
+    // If the user wants to send a private raw tx directly,
+    // web3.eth.sendRawPrivateTransaction should be used (which calls
+    // eth_sendRawPrivateTransaction)
+    const originalPayload = params.originalRequest?.params[0];
+    const privateFor = originalPayload?.privateFor;
+    const privateFrom = originalPayload?.privateFrom;
+    if (!privateFor && !privateFrom) {
+      return callback(null, params);
+    }
+
+    this.logger.trace(__(`Modifying blockchain '${params.request.method}' request:`));
+    this.logger.trace(__(`Original request data: ${JSON.stringify({ request: params.request, response: params.response })}`));
+
+    const { gas, gasPrice, gasLimit, from, to, value, nonce, bytecodeWithInitParam } = originalPayload;
+
+    try {
+      // Check if we have that account in our wallet
+      const account = accounts.find((acc) => Web3.utils.toChecksumAddress(acc.address) === Web3.utils.toChecksumAddress(from));
+      if (account && account.privateKey) {
+
+        const web3 = await this.web3;
+
+        // RawTransactionManager doesn't support websockets, and uses the
+        // currentProvider.host URI to communicate with the node over HTTP. Therefore, we can
+        // populate currentProvider.host with our proxy HTTP endpoint, without affecting
+        // web3's websocket connection.
+        // @ts-ignore
+        web3.eth.currentProvider.host = await this.events.request2("proxy:endpoint:http:get");
+
+        const rawTransactionManager = quorumjs.RawTransactionManager(web3, {
+          privateUrl: this.embark.config.blockchainConfig.clientConfig?.tesseraPrivateUrl ?? "http://localhost:9081"
+        });
+
+        const [data] = params.request.params;
+
+        const rawTx = {
+          gasPrice: this.stripPrefix(gasPrice) ?? (0).toString(16),
+          gasLimit: this.stripPrefix(gas) ?? this.stripPrefix(gasLimit) ?? (4300000).toString(16),
```

Comment:
Where does 4300000 come from?

Reply:
The `quorum-js` [example in the docs](https://github.com/jpmorganchase/quorum.js/blob/678c23733b3b5917a2e83d9185f92fe54bc891f8/7nodes-test/deployContractViaHttp.js#L89).